.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+
+updates:
+  - directory: "/"
```

Comment:
shouldn't this be `/scripts`?

Reply:
Does it work if it is `/scripts` ?
I have seen most projects do `/` hence put it like that.
But `/scripts` does make more sense

Reply:
we have python requirements only in `/scripts`, so that will only work I think.

Reply:
Ok!